Elevel/Application/Features/ApplicationUserFeatures/GetApplicationUserByIdQuery.cs

```diff
@@ -0,0 +1,71 @@
+﻿using AutoMapper;
+using Elevel.Application.Infrastructure;
+using Elevel.Application.Pagination;
+using Elevel.Domain.Models;
+using MediatR;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Elevel.Application.Features.ApplicationUserFeatures
+{
+    public class GetApplicationUserByIdQuery
+    {
+        public class Request : IRequest<Response>
+        {
+            public Guid Id { get; set; }
+        }
+
+        [Serializable]
+        public class Response
+        {
+            public Response(ApplicationUser user)
+            {
+
+                Id = user.Id;
+                FirstName = user.FirstName;
+                LastName = user.LastName;
+                CreationDate = user.CreationDate;
+                Avatar = user.Avatar;
+                Email = user.Email;
+                
+            }
+            public Guid Id { get; set; }
+            public string FirstName { get; set; }
+            public string LastName { get; set; }
+            public DateTimeOffset CreationDate { get; set; }
+            public string Avatar { get; set; }
+            public string Email { get; set; }
+        }
+
+        public class Handler : IRequestHandler<Request, Response>
+        {
+
+            private readonly IMapper _mapper;
```

Comment:
you should not keep variables that not used. but in current handler you should use it instead of manually copy each field. What will you tell me if you will have object with 100 props? will go and copy paste all 100 rows?